src/asammdf/blocks/mdf_v4.py

```diff
@@ -3042,30 +3129,30 @@ def append(
                         )
 
                     byte_size = 8
-                    kwargs = {
+                    cn_kwargs = {
                         "channel_type": v4c.CHANNEL_TYPE_VLSD,
                         "bit_count": 64,
                         "byte_offset": offset,
                         "bit_offset": 0,
                         "data_type": s_type,
-                        "data_block_addr": data_addr,
```

Comment:
must use the `data_addr` from the tempfile for the compacat VLDS channels

Reply:
Yes but I did not add it to ChannelKwargs because it is never accessed from kwargs, or did I miss something here? I can re-add it but as far as I can see it is never read from kwargs in Channel

Reply:
`group.signal_data` is used by `_load_signal_data` so setting the address in necessary at this point. The address will be updated if the file is saved